README.md

```
@@ -8,6 +8,19 @@ in AOSSIE projects or programs such as **Google Summer of Code (GSoC)**.
 
 ---
 
+## 📑 Table of Contents
+
+- [Who Should Use This Repository?](#-who-should-use-this-repository)
+- [Guidance for Contributors](#-guidance-for-contributors)
+- [Guidance for Mentors](#-guidance-for-mentors)
+- [Guidance for Admins](#-guidance-for-admins)
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)